### PR TITLE
ice driver backports from kernel 5.18 to 5.15.94.1 to enable baseline…

### PR DESCRIPTION
**Pull request by timothymiskell**
… switchdev functionality

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

ice driver backports from kernel 5.18 to 5.15.94.1 to enable baseline switchdev functionality prior to adding new switchdev related features

###### Change Log  
- Updated intel iidc
- Updated net-devlink
- Updated net-gtp
- Updated avf
- Updated ice driver

###### Does this affect the toolchain?  

NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Local build

### CODE REVIEW

**Comment by timothymiskell** [comment]
@microsoft-github-policy-service agree company="Intel"

**Review by rlmenge — CHANGES_REQUESTED** [review]
Please change target branch to `main`, the development branch for 2.0

File: SPECS/kernel-hci/kernel-hci.spec [hunk 1]
```diff
@@ -44,6 +44,11 @@ Patch22:        0023-net-mlx5-Bridge-rename-filter-fg-to-vlan_filter.patch
 Patch23:        0024-net-mlx5-Bridge-extract-VLAN-push-pop-actions-creati.patch
 Patch24:        0025-net-mlx5-Bridge-implement-infrastructure-for-VLAN-pr.patch
 Patch25:        0026-net-mlx5-Bridge-implement-QinQ-support.patch
+Patch26:        001-net-ice-core-netdevlink.patch
```
Comment (vinceaperri) [inline SPECS/kernel-hci/kernel-hci.spec:47]: Can you rename these patch files to `0027-net-ice-core-netdevlink.patch`, etc, so they are sorted properly in a `ls` output on the command line?

File: SPECS/kernel-hci/kernel-hci.spec [hunk 2]
```diff
@@ -8,8 +8,8 @@
 
 Summary:        Linux Kernel for HCI
 Name:           kernel-hci
-Version:        5.15.102.1
-Release:        1%{?dist}
+Version:        5.15.94.1
```
Comment (vinceaperri) [inline SPECS/kernel-hci/kernel-hci.spec:11]: You may need to rebase on top of main. Only the Release should be bumped.

File: SPECS/kernel-hci/kernel-hci.spec [hunk 3]
```diff
@@ -418,15 +428,6 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_sysconfdir}/bash_completion.d/bpftool
 
 %changelog
```
Comment (vinceaperri) [inline SPECS/kernel-hci/kernel-hci.spec:430]: Please add to the changelog documenting what this change does.

**Pull request closed without merging**